source/macros.tex

```diff
@@ -160,7 +160,7 @@
 \newcommand{\defnx}[2]{\indexdefn{#2}\textit{#1}\xspace}
 \newcommand{\defn}[1]{\defnx{#1}{#1}}
 \newcommand{\term}[1]{\textit{#1}\xspace}
-\newcommand{\grammarterm}[1]{\textit{#1}\xspace}
+\newcommand{\grammarterm}[1]{\textit{#1}}
```

Comment:
Ah, OK :)

Have you checked that this does not cause any change in formatting anywhere else?